entrypoint.sh

```diff
@@ -159,6 +169,34 @@ function get_files_matching_filetype() {
 	echo "${matching_files[@]}"
 }
 
+function load_filetype_array() {
+	local val=$1
+	declare -a types
+
+	val=$(echo "$val" | jq -r '.[]')
+	while IFS= read -r filetype; do
+		types+=("$filetype")
+	done <<<"$val"
+
+	echo "${types[@]}"
+}
+
+function has_autofix() {
+	local lint_name=$1
+
+	while IFS= read -r line; do
+		name=$(echo "$line" | jq -r ".name")
+		if [[ "$name" == "$lint_name" ]]; then
+			if echo "$line" | jq -e '.autofix' > /dev/null; then
+				# If linter has an autofix exit successfully
+				return 0
+			else
+				return 1
+			fi
+		fi
+	done < <(jq -c '.[]' $input)
```

Comment:
Need to quote this in case it has spaces.  Also, it makes linters happy
```suggestion
	done < <(jq -c '.[]' "${input}")
```